gson/src/main/java/com/google/gson/FieldNamingStrategy.java

```diff
@@ -37,4 +40,15 @@ public interface FieldNamingStrategy {
    * @since 1.3
    */
   public String translateName(Field f);
+
+  /**
+   * Translates the field name into its JSON field alternative names representation. used for
+   * deserialization only. This is similar to {@link SerializedName#alternate()}.
+   *
+   * @param f the field object that we are translating
+   * @return the list of possible translated field names.
+   */
```

Comment:
Sorry, forgot to mention that this should have a `@since $next-version$`
(`$next-version$` is a special placeholder which is replaced on next release)